controller/appcontroller.go

```diff
@@ -778,34 +843,15 @@ func (ctrl *ApplicationController) refreshAppConditions(app *appv1.Application)
 			conditions = append(conditions, specConditions...)
 		}
 	}
-
-	// List of condition types which have to be reevaluated by controller; all remaining conditions should stay as is.
-	reevaluateTypes := map[appv1.ApplicationConditionType]bool{
```

Comment:
what's happend to re-evaluation?

Reply:
any thoughts on this?

Reply:
The `SetConditions` does it now. The method keep condition untouched unless it is specified in `evaluatedTypes` parameter